buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy

```diff
@@ -34,7 +34,7 @@ abstract class AndroidBuckRuleComposer extends JvmBuckRuleComposer {
         return "app_lib_${target.name}"
     }
 
-    static String bin(AndroidAppTarget target) {
+    static String bin(AndroidTarget target) {
```

Comment:
Having `bin_` for android libraries does not make sense. Lets name these differently since the `bin_` is supposed to be for actual apps.

Lets call these rules with name `lib_instrumentation_${target.name}`

Reply:
This `bin` method is called on [here](https://github.com/uber/okbuck/blob/master/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBinaryRuleComposer.groovy#L63).
Though of two solutions, either passing a possibly custom name for this composer, or create a function named bin, which accepts the androidLibInstrumentation target, that will output the `lib_instrumentation` name